File: FirmaTransport/FirmaTransport/domain/Cursa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FirmaTransport.domain
{
    class Cursa : Entity<int>
    {
        public string destinatie { get; set; }
        public DateTime plecare { get; set; }
        public int locuriDisponibile { get; set; }

        public Cursa(int id, string destinatie, DateTime plecare, int locuriDisponibile)
        {
            this.id = id;
            this.destinatie = destinatie;
            this.plecare = plecare;
            this.locuriDisponibile = locuriDisponibile;
        }

        public override string ToString()
        {
            return id + " " + destinatie + " " + plecare + " " + locuriDisponibile;
        }
    }
}

File: FirmaTransportSocketsJava/Model/src/main/java/domain/Cursa.java
package domain;


import java.time.LocalDateTime;


public class Cursa extends Entity<Integer> {
    private String destinatie;
    private LocalDateTime plecare;
    private Integer locuriDisponibile;


    public Cursa(String destinatie, LocalDateTime plecare, Integer locuriDisponibile) {
        this.destinatie = destinatie;
        this.plecare = plecare;
        this.locuriDisponibile = locuriDisponibile;
    }

    public String getDestinatie() {
        return destinatie;
    }

    public void setDestinatie(String destinatie) {
        this.destinatie = destinatie;
    }

    public LocalDateTime getPlecare() {
        return plecare;
    }

    public void setPlecare(LocalDateTime plecare) {
        this.plecare = plecare;
    }

    public Integer getLocuriDisponibile() {
        return locuriDisponibile;
    }

    public void setLocuriDisponibile(Integer locuriDisponibile) {
        this.locuriDisponibile = locuriDisponibile;
    }



    @Override
    public String toString() {
        return "Cursa{" +
                "destinatie='" + destinatie + '\'' +
                ", plecare=" + plecare +
                ", locuriDisponibile=" + locuriDisponibile +
                ", id=" + id +
                '}';
    }
}
